processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
--- memsim.hex	2024-02-12 19:58:22.836306532 -0500
+++ memcpu.hex	2024-02-12 19:58:22.812305627 -0500
@@ -13,7 +13,7 @@
 :040020000000733732
 :0400210000002701B3
 :040022000000133790
-:04002300DEADBEEFA1
+:04002300DEAD00004E
 :04003C000000733716
 :04003D000000270197
 :04003E000000133774
processing file asmFiles/test.rtype.asm (5 of 5)
--- memsim.hex	2024-02-12 19:58:24.263360353 -0500
+++ memcpu.hex	2024-02-12 19:58:24.243359599 -0500
@@ -15,5 +15,4 @@
 :04000E00AEAB002075
 :04000F00AEAC00246F
 :04001000FFFFFFFFF0
-:043FC0000000F7F90D
 :00000001FF
processing file asmFiles/add.asm (1 of 19)
--- memsim.hex	2024-02-12 20:36:08.851728376 -0500
+++ memcpu.hex	2024-02-12 20:36:08.835727775 -0500
@@ -7,5 +7,4 @@
 :04000C00AC830000C1
 :040012008C850000D9
 :04001800FFFFFFFFE8
-:043FC20000000009F2
 :00000001FF
processing file asmFiles/branch.asm (2 of 19)
processing file asmFiles/test.branchjump1.asm (1 of 5)
--- memsim.hex	2024-02-12 20:36:17.317046501 -0500
+++ memcpu.hex	2024-02-12 20:36:17.302045937 -0500
@@ -13,6 +13,4 @@
 :04000C00AC41000CF7
 :04000D0017E3FFF9FD
 :04000E00FFFFFFFFF2
-:0400220000000018C2
-:0400240000000030A8
 :00000001FF
processing file asmFiles/test.ldst1.asm (2 of 5)
--- memsim.hex	2024-02-12 20:36:18.500090960 -0500
+++ memcpu.hex	2024-02-12 20:36:18.485090396 -0500
@@ -15,7 +15,7 @@
 :04003C000000733716
 :04003D000000270197
 :04003E000000133774
-:040040000000733712
-:040041000000270193
-:040042000000133770
+:0400400000000400B8
+:040041000000733711
+:040042000000270192
 :00000001FF
processing file asmFiles/test.ldst2.asm (3 of 5)
--- memsim.hex	2024-02-12 20:36:19.592131998 -0500
+++ memcpu.hex	2024-02-12 20:36:19.578131472 -0500
@@ -10,10 +10,9 @@
 :04000C00AC450008F7
 :04000D00AC4A000CED
 :04000E00FFFFFFFFF2
-:040200000000733750
-:0402010000002701D1
-:0402020000001337AE
-:04020300FEEDBEEF5F
+:04020100000073374F
+:0402020000002701D0
+:0402030000001337AD
 :0403C000000073378F
 :0403C1000000270110
 :0403C20000001337ED
processing file asmFiles/test.loadstore.asm (4 of 5)
--- memsim.hex	2024-02-12 20:36:20.691173300 -0500
+++ memcpu.hex	2024-02-12 20:36:20.676172736 -0500
@@ -10,10 +10,9 @@
 :04000900AC450008FA
 :04000A00AC47000CF3
 :04000B00FFFFFFFFF5
-:040020000000733732
-:0400210000002701B3
-:040022000000133790
-:04002300DEADBEEFA1
+:040021000000733731
+:0400220000002701B2
+:04002300000013378F
 :04003C000000733716
 :04003D000000270197
 :04003E000000133774
processing file asmFiles/test.rtype.asm (5 of 5)
--- memsim.hex	2024-02-12 20:36:21.854217007 -0500
+++ memcpu.hex	2024-02-12 20:36:21.839216443 -0500
@@ -28,14 +28,13 @@
 :04001B00AEAC002463
 :04001C00FFFFFFFFE4
 :043F0000FFFF0806B1
-:043FC0000000F7F90D
-:043FC1000000126189
-:043FC20000000009F2
-:043FC30000010A5A95
-:043FC40000007F3941
-:043FC500FFFFDA70B0
-:043FC600000037F8C8
-:043FC700000021567F
-:043FC800000D269032
-:043FC900000006935B
+:043FC1000000F7F90C
+:043FC2000000126188
+:043FC30000000009F1
+:043FC40000010A5A94
+:043FC50000007F3940
+:043FC600FFFFDA70AF
+:043FC700000037F8C7
+:043FC800000021567E
+:043FC900000D269031
 :00000001FF
processing file asmFiles/add.asm (1 of 19)
processing file asmFiles/test.branchjump1.asm (1 of 5)
--- memsim.hex	2024-02-12 20:38:46.805543617 -0500
+++ memcpu.hex	2024-02-12 20:38:46.784542825 -0500
@@ -1,10 +1,8 @@
-:040000003401BA5CB1
 :040001003402008045
 :04000200340F003483
 :0400030010000001E8
 :04000400AC4100000B
 :040005000C000009E2
-:04000600AC41000405
 :04000700AC5F0010DA
 :04000800FFFFFFFFF8
 :04000900001F182597
@@ -13,6 +11,4 @@
 :04000C00AC41000CF7
 :04000D0017E3FFF9FD
 :04000E00FFFFFFFFF2
-:0400220000000018C2
-:0400240000000030A8
 :00000001FF
processing file asmFiles/test.ldst1.asm (2 of 5)
--- memsim.hex	2024-02-12 20:38:47.941586451 -0500
+++ memcpu.hex	2024-02-12 20:38:47.926585886 -0500
@@ -15,7 +15,6 @@
 :04003C000000733716
 :04003D000000270197
 :04003E000000133774
-:040040000000733712
-:040041000000270193
-:040042000000133770
+:040041000000733711
+:040180000000040077
 :00000001FF
processing file asmFiles/test.ldst2.asm (3 of 5)
--- memsim.hex	2024-02-12 20:38:49.119630245 -0500
+++ memcpu.hex	2024-02-12 20:38:49.096629392 -0500
@@ -1,4 +1,3 @@
-:040000003C0AFEEDCB
 :0400010034010F00B7
 :0400020034020800BC
 :04000300354ABEEFCD
@@ -10,10 +9,9 @@
 :04000C00AC450008F7
 :04000D00AC4A000CED
 :04000E00FFFFFFFFF2
-:040200000000733750
-:0402010000002701D1
-:0402020000001337AE
-:04020300FEEDBEEF5F
+:040200003C0AFEEDC9
+:04020100000073374F
+:0402020000002701D0
 :0403C000000073378F
 :0403C1000000270110
 :0403C20000001337ED
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/add.asm (1 of 19)
processing file asmFiles/branch.asm (2 of 19)
processing file asmFiles/count_days.asm (3 of 19)
processing file asmFiles/hazards.control.asm (1 of 2)
processing file asmFiles/hazards.data.asm (2 of 2)
--- memsim.hex	2024-02-12 20:44:54.518274163 -0500
+++ memcpu.hex	2024-02-12 20:44:54.499273451 -0500
@@ -1,4 +1,4 @@
-:040000000000000AF2
+:04000000FFFFFFFD02
 :0400010020030003D5
 :0400020020040004D2
 :0400030020050005CF
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
--- memsim.hex	2024-02-12 20:45:19.056194253 -0500
+++ memcpu.hex	2024-02-12 20:45:19.033193391 -0500
@@ -27,7 +27,6 @@
 :04001A00AEAB002069
 :04001B00AEAC002463
 :04001C00FFFFFFFFE4
-:043F0000FFFF0806B1
 :043FC0000000F7F90D
 :043FC1000000126189
 :043FC20000000009F2
processing file asmFiles/add.asm (1 of 20)
processing file asmFiles/branch.asm (2 of 20)
processing file asmFiles/count_days.asm (3 of 20)
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
processing file asmFiles/hazards.control.asm (1 of 3)
processing file asmFiles/hazards.data.asm (2 of 3)
processing file asmFiles/hazards.load.asm (3 of 3)
processing file asmFiles/add.asm (1 of 20)
processing file asmFiles/branch.asm (2 of 20)
processing file asmFiles/count_days.asm (3 of 20)
processing file asmFiles/slt.asm (1 of 1)
--- memsim.hex	2024-02-16 22:00:46.513267146 -0500
+++ memcpu.hex	2024-02-16 22:00:46.498266582 -0500
@@ -8,5 +8,5 @@
 :040007008C26000043
 :040008008C2700043D
 :04000900FFFFFFFFF7
-:043FC00000000001FC
+:043FC1000000FF00FD
 :00000001FF
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/jump2.asm (1 of 2)
processing file asmFiles/jump.asm (2 of 2)
--- memsim.hex	2024-02-16 22:01:11.171193307 -0500
+++ memcpu.hex	2024-02-16 22:01:11.155192706 -0500
@@ -18,7 +18,5 @@
 :04001100AC7F0000C0
 :040012000800000BD7
 :04001300AC620004D7
-:043FC00000000028D5
 :043FC1000000AAAAA8
-:043FC2000000AAAAA7
 :00000001FF
processing file asmFiles/mult.asm (1 of 1)
--- memsim.hex	2024-02-16 22:01:23.470655279 -0500
+++ memcpu.hex	2024-02-16 22:01:23.456654754 -0500
@@ -13,5 +13,5 @@
 :04000C0027BDFFFC11
 :04000D00AFAB000095
 :04000E00FFFFFFFFF2
-:043FFE0000000006B9
+:043FFE000000FFF8C8
 :00000001FF
processing file asmFiles/mult_procedure.asm (1 of 1)
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
processing file asmFiles/branch.asm (1 of 1)
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/jump2.asm (1 of 2)
processing file asmFiles/jump.asm (2 of 2)
--- memsim.hex	2024-02-16 22:52:18.154405463 -0500
+++ memcpu.hex	2024-02-16 22:52:18.136404787 -0500
@@ -18,7 +18,5 @@
 :04001100AC7F0000C0
 :040012000800000BD7
 :04001300AC620004D7
-:043FC00000000028D5
 :043FC1000000AAAAA8
-:043FC2000000AAAAA7
 :00000001FF
processing file asmFiles/test.branchjump1.asm (1 of 5)
--- memsim.hex	2024-02-17 13:08:56.240156258 -0500
+++ memcpu.hex	2024-02-16 22:52:53.321726457 -0500
@@ -1,18 +1,22 @@
-:040000003401BA5CB1
-:040001003402008045
-:04000200340F003483
-:0400030010000001E8
-:04000400AC4100000B
-:040005000C000009E2
-:04000600AC41000405
-:04000700AC5F0010DA
-:04000800FFFFFFFFF8
-:04000900001F182597
-:04000A00AC5F0008DF
-:04000B000C00000DD8
-:04000C00AC41000CF7
-:04000D0017E3FFF9FD
+:0400000033BD00000C
+:0400010037BDFFFC0C
+:0400020030210000A9
+:040003003421AAAA50
+:040004003042000086
+:0400050034420BADC9
+:040006003063000063
+:040007003463FF005F
+:040008003405FF08B4
+:040009000C000011D6
+:04000A000060102062
+:04000B00AC610004E0
+:04000C0023E40014D5
+:04000D000080000867
 :04000E00FFFFFFFFF2
-:0400220000000018C2
-:0400240000000030A8
+:04000F00AC610008D8
+:04001000FFFFFFFFF0
+:04001100AC7F0000C0
+:040012000800000BD7
+:04001300AC620004D7
+:043FC1000000AAAAA8
 :00000001FF
processing file asmFiles/test.ldst1.asm (2 of 5)
--- memsim.hex	2024-02-17 13:08:56.520166730 -0500
+++ memcpu.hex	2024-02-16 22:52:53.321726457 -0500
@@ -1,21 +1,22 @@
-:04000000340100F0D7
-:0400010034020100C4
-:0400020034030200C1
-:0400030034040300BE
-:0400040034050400BB
-:040005008C26000045
-:040006008C2700043F
-:040007008C28000839
-:0400080034040500B7
-:0400090034050600B4
-:04000A00AC46000000
-:04000B00AC470004FA
-:04000C00AC480008F4
-:04000D00FFFFFFFFF3
-:04003C000000733716
-:04003D000000270197
-:04003E000000133774
-:040040000000733712
-:040041000000270193
-:040042000000133770
+:0400000033BD00000C
+:0400010037BDFFFC0C
+:0400020030210000A9
+:040003003421AAAA50
+:040004003042000086
+:0400050034420BADC9
+:040006003063000063
+:040007003463FF005F
+:040008003405FF08B4
+:040009000C000011D6
+:04000A000060102062
+:04000B00AC610004E0
+:04000C0023E40014D5
+:04000D000080000867
+:04000E00FFFFFFFFF2
+:04000F00AC610008D8
+:04001000FFFFFFFFF0
+:04001100AC7F0000C0
+:040012000800000BD7
+:04001300AC620004D7
+:043FC1000000AAAAA8
 :00000001FF
processing file asmFiles/test.ldst2.asm (3 of 5)
--- memsim.hex	2024-02-17 13:08:56.900180940 -0500
+++ memcpu.hex	2024-02-16 22:52:53.321726457 -0500
@@ -1,20 +1,22 @@
-:040000003C0AFEEDCB
-:0400010034010F00B7
-:0400020034020800BC
-:04000300354ABEEFCD
-:040005008C23000048
-:040006008C24000442
-:040007008C2500083C
-:04000A00AC43000003
-:04000B00AC440004FD
-:04000C00AC450008F7
-:04000D00AC4A000CED
+:0400000033BD00000C
+:0400010037BDFFFC0C
+:0400020030210000A9
+:040003003421AAAA50
+:040004003042000086
+:0400050034420BADC9
+:040006003063000063
+:040007003463FF005F
+:040008003405FF08B4
+:040009000C000011D6
+:04000A000060102062
+:04000B00AC610004E0
+:04000C0023E40014D5
+:04000D000080000867
 :04000E00FFFFFFFFF2
-:040200000000733750
-:0402010000002701D1
-:0402020000001337AE
-:04020300FEEDBEEF5F
-:0403C000000073378F
-:0403C1000000270110
-:0403C20000001337ED
+:04000F00AC610008D8
+:04001000FFFFFFFFF0
+:04001100AC7F0000C0
+:040012000800000BD7
+:04001300AC620004D7
+:043FC1000000AAAAA8
 :00000001FF
processing file asmFiles/test.loadstore.asm (4 of 5)
--- memsim.hex	2024-02-17 13:08:57.281195189 -0500
+++ memcpu.hex	2024-02-16 22:52:53.321726457 -0500
@@ -1,20 +1,22 @@
-:04000000340100F0D7
-:040001003402008045
-:040002003C07DEAD2C
-:0400030034E7BEEF31
-:040004008C23000049
-:040005008C24000443
-:040006008C2500083D
-:04000700AC43000006
-:04000800AC44000400
-:04000900AC450008FA
-:04000A00AC47000CF3
-:04000B00FFFFFFFFF5
-:040020000000733732
-:0400210000002701B3
-:040022000000133790
-:04002300DEADBEEFA1
-:04003C000000733716
-:04003D000000270197
-:04003E000000133774
+:0400000033BD00000C
+:0400010037BDFFFC0C
+:0400020030210000A9
+:040003003421AAAA50
+:040004003042000086
+:0400050034420BADC9
+:040006003063000063
+:040007003463FF005F
+:040008003405FF08B4
+:040009000C000011D6
+:04000A000060102062
+:04000B00AC610004E0
+:04000C0023E40014D5
+:04000D000080000867
+:04000E00FFFFFFFFF2
+:04000F00AC610008D8
+:04001000FFFFFFFFF0
+:04001100AC7F0000C0
+:040012000800000BD7
+:04001300AC620004D7
+:043FC1000000AAAAA8
 :00000001FF
processing file asmFiles/test.rtype.asm (5 of 5)
--- memsim.hex	2024-02-17 13:08:57.685210298 -0500
+++ memcpu.hex	2024-02-16 22:52:53.321726457 -0500
@@ -1,41 +1,22 @@
-:040000003401D2698C
-:04000100340237F19D
-:040002003415FF00B2
-:040003003416FC00B3
-:040004000022182599
-:040005000022202491
-:040006003025000F92
-:040007000022302182
-:0400080024678740A2
-:04000900008240230E
-:04000A0000A24826E2
-:04000B00382AF33F5D
-:04000C00340E0004AA
-:04000D0001C15804D1
-:04000E00340E0005A7
-:04000F0001C16006C5
-:04001000002268273B
-:04001100AECD000070
-:04001200AEA3000099
-:04001300AEA4000493
-:04001400AEA500088D
-:04001500AEA6000C87
-:04001600AEA7001081
-:04001700AEA800147B
-:04001800AEA9001875
-:04001900AEAA001C6F
-:04001A00AEAB002069
-:04001B00AEAC002463
-:04001C00FFFFFFFFE4
-:043F0000FFFF0806B1
-:043FC0000000F7F90D
-:043FC1000000126189
-:043FC20000000009F2
-:043FC30000010A5A95
-:043FC40000007F3941
-:043FC500FFFFDA70B0
-:043FC600000037F8C8
-:043FC700000021567F
-:043FC800000D269032
-:043FC900000006935B
+:0400000033BD00000C
+:0400010037BDFFFC0C
+:0400020030210000A9
+:040003003421AAAA50
+:040004003042000086
+:0400050034420BADC9
+:040006003063000063
+:040007003463FF005F
+:040008003405FF08B4
+:040009000C000011D6
+:04000A000060102062
+:04000B00AC610004E0
+:04000C0023E40014D5
+:04000D000080000867
+:04000E00FFFFFFFFF2
+:04000F00AC610008D8
+:04001000FFFFFFFFF0
+:04001100AC7F0000C0
+:040012000800000BD7
+:04001300AC620004D7
+:043FC1000000AAAAA8
 :00000001FF
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
processing file asmFiles/add.asm (1 of 20)
processing file asmFiles/branch.asm (2 of 20)
processing file asmFiles/count_days.asm (3 of 20)
processing file asmFiles/hazards.control.asm (1 of 3)
processing file asmFiles/hazards.data.asm (2 of 3)
processing file asmFiles/hazards.load.asm (3 of 3)
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
processing file asmFiles/hazards.control.asm (1 of 3)
processing file asmFiles/hazards.data.asm (2 of 3)
processing file asmFiles/hazards.load.asm (3 of 3)
processing file asmFiles/add.asm (1 of 20)
processing file asmFiles/branch.asm (2 of 20)
processing file asmFiles/count_days.asm (3 of 20)
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2024-02-17 13:12:04.170184095 -0500
+++ memcpu.hex	2024-02-17 13:12:04.153183459 -0500
@@ -12,27 +12,16 @@
 :04000B0015E0FFF904
 :04000C00FFFFFFFFF4
 :0400210000000001DA
-:0400220000000001D9
 :0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400250000000004D3
+:0400270000000008CD
+:0400290000000010C3
+:04002B0000000020B1
+:04002D00000000408F
+:04002F00000000804D
+:0400310000000100CA
+:0400330000000200C7
+:0400350000000400C3
+:0400370000000800BD
 :0403C0000000001623
 :00000001FF
processing file asmFiles/jump2.asm (1 of 2)
processing file asmFiles/jump.asm (2 of 2)
--- memsim.hex	2024-02-17 13:13:06.929531044 -0500
+++ memcpu.hex	2024-02-17 13:13:06.910530334 -0500
@@ -18,7 +18,5 @@
 :04001100AC7F0000C0
 :040012000800000BD7
 :04001300AC620004D7
-:043FC00000000028D5
 :043FC1000000AAAAA8
-:043FC2000000AAAAA7
 :00000001FF
processing file asmFiles/mult.asm (1 of 2)
--- memsim.hex	2024-02-17 13:13:19.630005987 -0500
+++ memcpu.hex	2024-02-17 13:13:19.606005090 -0500
@@ -13,5 +13,5 @@
 :04000C0027BDFFFC11
 :04000D00AFAB000095
 :04000E00FFFFFFFFF2
-:043FFE0000000006B9
+:043FFE000000FFF8C8
 :00000001FF
processing file asmFiles/mult_procedure.asm (2 of 2)
processing file asmFiles/jump2.asm (1 of 2)
processing file asmFiles/jump.asm (2 of 2)
--- memsim.hex	2024-02-17 13:19:01.925806391 -0500
+++ memcpu.hex	2024-02-17 13:19:01.904805606 -0500
@@ -18,7 +18,5 @@
 :04001100AC7F0000C0
 :040012000800000BD7
 :04001300AC620004D7
-:043FC00000000028D5
 :043FC1000000AAAAA8
-:043FC2000000AAAAA7
 :00000001FF
processing file asmFiles/add.asm (1 of 19)
processing file asmFiles/branch.asm (2 of 19)
processing file asmFiles/count_days.asm (3 of 19)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
processing file asmFiles/test.branchjump1.asm (1 of 5)
processing file asmFiles/test.ldst1.asm (2 of 5)
processing file asmFiles/test.ldst2.asm (3 of 5)
processing file asmFiles/test.loadstore.asm (4 of 5)
processing file asmFiles/test.rtype.asm (5 of 5)
processing file asmFiles/hazards.control.asm (1 of 3)
processing file asmFiles/hazards.data.asm (2 of 3)
processing file asmFiles/hazards.load.asm (3 of 3)
processing file asmFiles/slt.asm (1 of 1)
processing file asmFiles/fib.asm (1 of 1)
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/mult_procedure.asm (1 of 1)
processing file asmFiles/add.asm (1 of 19)
--- memsim.hex	2024-02-17 14:43:58.987429969 -0500
+++ memcpu.hex	2024-02-17 14:34:48.643848339 -0500
@@ -1,11 +1,120 @@
-:0400000000200824B0
-:0400010034210003A3
-:040002000040102486
-:04000300344200067D
-:04000700002218209B
-:040008003404FF08B5
-:04000C00AC830000C1
-:040012008C850000D9
-:04001800FFFFFFFFE8
-:043FC20000000009F2
+:04000000341D00802B
+:0400010034010001C5
+:0400020034020004C0
+:04000300AFA00000AA
+:040004008FA30004C2
+:040005008FA40008BC
+:0400060027A5000C1E
+:040007008CA60000C3
+:0400080000C33823D6
+:0400090010E00004FF
+:04000A0000A2282107
+:04000B00008120232D
+:04000C00108000025E
+:04000D001000FFF9E7
+:04000E00AFA500009A
+:04000F00FFFFFFFFF1
+:0400200000000124B7
+:0400210000005C6F10
+:040022000000006476
+:040023000000087D54
+:0400240000005FCBAE
+:040025000000A41A19
+:04002600000041098C
+:04002700000045226E
+:040028000000700F55
+:040029000000766DF0
+:04002A0000006F6003
+:04002B0000008A5EE9
+:04002C0000009580BB
+:04002D00000070A3BC
+:04002E000000AEA977
+:04002F000000711A42
+:0400300000006F81DC
+:0400310000008F9AA2
+:040032000000258421
+:040033000000A5998B
+:040034000000401573
+:040035000000CE8178
+:040036000000F55B76
+:040037000000399EEE
+:040038000000A23FE3
+:040039000000358806
+:04003A00000033ACE3
+:04003B000000BCE71E
+:04003C0000002A6B2B
+:04003D0000009FA17F
+:04003E000000C94BAA
+:04003F000000C65B9C
+:040040000000006854
+:040041000000F4992E
+:0400420000005F71EA
+:040043000000D06F7A
+:04004400000014DFC5
+:040045000000116541
+:040046000000F88D31
+:0400470000004BA4C6
+:0400480000002E7412
+:0400490000005C6FE8
+:04004A000000D11EC3
+:04004B0000009222FD
+:04004C000000ACDB29
+:04004D000000103867
+:04004E000000AB17EC
+:04004F000000F7CEE8
+:0400500000008A9E84
+:0400510000009AA36E
+:040052000000B49561
+:0400530000008A5EC1
+:040054000000D85977
+:0400550000000BACF0
+:040056000000D0DBFB
+:04005700000035521E
+:040058000000A6B04E
+:040059000000727FB2
+:04005A00000028E496
+:04005B000000E5CFED
+:04005C000000163C4E
+:04005D00000034115A
+:04005E0000008F0708
+:04005F000000FAB7EC
+:0400600000000F3459
+:040061000000DABF02
+:0400620000006F6FBC
+:040063000000C5983C
+:040064000000F4960E
+:0400650000009A9A63
+:040066000000BD6A6F
+:04006700000021363E
+:040068000000810A09
+:040069000000CA5574
+:04006A0000008BCE39
+:04006B0000002AC4A3
+:04006C000000DDCEE5
+:04006D000000DD06AC
+:04006E000000C4FCCE
+:04006F000000FB2F63
+:040070000000EE5F3F
+:040071000000FD305E
+:040072000000C54085
+:040073000000D5F1C3
+:040074000000BDAD1E
+:04007500000045C37F
+:040076000000708A8C
+:040077000000A35989
+:040078000000F40D83
+:040079000000BA06C3
+:04007A000000BACEFA
+:04007B000000B44786
+:04007C0000003F48F9
+:04007D000000899E58
+:04007E00000080847A
+:04007F000000BDB907
+:040080000000A05A82
+:040081000000E22574
+:040082000000FB0C73
+:040083000000B2B215
+:040084000000A4DBF9
+:0400850000008BF9F3
+:04008600000012F76D
 :00000001FF
processing file asmFiles/branch.asm (2 of 19)
--- memsim.hex	2024-02-17 14:44:00.607490556 -0500
+++ memcpu.hex	2024-02-17 14:34:48.643848339 -0500
@@ -1,25 +1,120 @@
-:0400000033BD00000C
-:0400010037BDFFFC0C
-:0400020030210000A9
-:040003003421AAAA50
-:040004003042000086
-:0400050034420BADC9
-:040006003063000063
-:040007003463FF005F
-:040008003084000040
-:040009003484FF0438
-:04000A0010220004BC
-:04000B0014220004B7
-:04000C002042FFFF90
-:04000D0014220004B5
-:04000E0010220002BA
-:04000F00AC620000DF
-:04001000AC610000DF
-:04001100AC820000BD
-:04001200AC810000BD
-:040013008C650000F8
-:040014008C860000D6
-:04001500FFFFFFFFEB
-:043FC0000000AAAAA9
-:043FC1000000AAAAA8
+:04000000341D00802B
+:0400010034010001C5
+:0400020034020004C0
+:04000300AFA00000AA
+:040004008FA30004C2
+:040005008FA40008BC
+:0400060027A5000C1E
+:040007008CA60000C3
+:0400080000C33823D6
+:0400090010E00004FF
+:04000A0000A2282107
+:04000B00008120232D
+:04000C00108000025E
+:04000D001000FFF9E7
+:04000E00AFA500009A
+:04000F00FFFFFFFFF1
+:0400200000000124B7
+:0400210000005C6F10
+:040022000000006476
+:040023000000087D54
+:0400240000005FCBAE
+:040025000000A41A19
+:04002600000041098C
+:04002700000045226E
+:040028000000700F55
+:040029000000766DF0
+:04002A0000006F6003
+:04002B0000008A5EE9
+:04002C0000009580BB
+:04002D00000070A3BC
+:04002E000000AEA977
+:04002F000000711A42
+:0400300000006F81DC
+:0400310000008F9AA2
+:040032000000258421
+:040033000000A5998B
+:040034000000401573
+:040035000000CE8178
+:040036000000F55B76
+:040037000000399EEE
+:040038000000A23FE3
+:040039000000358806
+:04003A00000033ACE3
+:04003B000000BCE71E
+:04003C0000002A6B2B
+:04003D0000009FA17F
+:04003E000000C94BAA
+:04003F000000C65B9C
+:040040000000006854
+:040041000000F4992E
+:0400420000005F71EA
+:040043000000D06F7A
+:04004400000014DFC5
+:040045000000116541
+:040046000000F88D31
+:0400470000004BA4C6
+:0400480000002E7412
+:0400490000005C6FE8
+:04004A000000D11EC3
+:04004B0000009222FD
+:04004C000000ACDB29
+:04004D000000103867
+:04004E000000AB17EC
+:04004F000000F7CEE8
+:0400500000008A9E84
+:0400510000009AA36E
+:040052000000B49561
+:0400530000008A5EC1
+:040054000000D85977
+:0400550000000BACF0
+:040056000000D0DBFB
+:04005700000035521E
+:040058000000A6B04E
+:040059000000727FB2
+:04005A00000028E496
+:04005B000000E5CFED
+:04005C000000163C4E
+:04005D00000034115A
+:04005E0000008F0708
+:04005F000000FAB7EC
+:0400600000000F3459
+:040061000000DABF02
+:0400620000006F6FBC
+:040063000000C5983C
+:040064000000F4960E
+:0400650000009A9A63
+:040066000000BD6A6F
+:04006700000021363E
+:040068000000810A09
+:040069000000CA5574
+:04006A0000008BCE39
+:04006B0000002AC4A3
+:04006C000000DDCEE5
+:04006D000000DD06AC
+:04006E000000C4FCCE
+:04006F000000FB2F63
+:040070000000EE5F3F
+:040071000000FD305E
+:040072000000C54085
+:040073000000D5F1C3
+:040074000000BDAD1E
+:04007500000045C37F
+:040076000000708A8C
+:040077000000A35989
+:040078000000F40D83
+:040079000000BA06C3
+:04007A000000BACEFA
+:04007B000000B44786
+:04007C0000003F48F9
+:04007D000000899E58
+:04007E00000080847A
+:04007F000000BDB907
+:040080000000A05A82
+:040081000000E22574
+:040082000000FB0C73
+:040083000000B2B215
+:040084000000A4DBF9
+:0400850000008BF9F3
+:04008600000012F76D
 :00000001FF
processing file asmFiles/count_days.asm (3 of 19)
processing file asmFiles/add.asm (1 of 19)
processing file asmFiles/branch.asm (2 of 19)
processing file asmFiles/count_days.asm (3 of 19)
